docker/app/Dockerfile

```diff
@@ -0,0 +1,90 @@
+# syntax=docker/dockerfile:1

```

Comment:
This Dockerfile builds a container image for the Gumroad app with all required dependencies.
It will use three volumes: one for Ruby gems, one for Node.js dependencies, and one mounted to /workspace so code changes are reflected in real time without breaking auto-refresh.